src/tests/ftest/security/cont_owner.py

```diff
@@ -125,3 +137,56 @@ def test_container_set_owner_no_check_posix(self):
                                               no_check=True)
         self.verify_daos_pool_cont_result(result, "set owner with uid/gid with no-check",
                                           "pass", None)
+
+    def test_dmg_cont_set_owner(self):
+        """
+        Description:
+            Verify dmg container set-owner
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=security,control,container,cont_sec,cont_set_owner
+        :avocado: tags=DaosContainerOwnerTest
+        :avocado: tags=test_dmg_cont_set_owner
+        """
+        test_user = "fakeuser@"
+        test_group = "fakegroup@"
+
+        self._create_cont_with_acl(cont_type="python")
+        orig_owner = self._get_ownership()
+
+        # No user/group
+        with self.dmg.no_exception():
+            result = self.get_dmg_command().cont_set_owner(self.pool.uuid, self.container.uuid)
```

Comment:
Since the command supports uuid and label, you could use `identifier` here, which ends up using the label by default. And similar for almost anywhere else using `.uuid`
```suggestion
            result = self.get_dmg_command().cont_set_owner(
                self.pool.identifier, self.container.identifier)
```

Reply:
Oh neat! I'll update that if I need to push again. I wanted to test both uuid and label, but in most of these cases it really doesn't matter which is used.

Reply:
The general idea is to use labels for everything unless we specifically want to test UUIDs

Reply:
Fixed this -- I used the generic identifier for most cases, and explicitly used the label and UUID for the last two successful cases, to make sure both work.